### DISCUSSION TITLE
Pass more Tesseract parameters to `get_textpage_ocr`

### DISCUSSION THREAD

**Discussion by BoucheDimitri** [pymupdf/PyMuPDF#3918] [Q&A]
Hello, 

I was wondering about the function `get_textpage_ocr`. It supports chosing the language and the DPI. But what about other Tesseract parameters ? Is there was a way to pass more of these to `get_textpage_ocr` such as
- Page segmentation method (--psm parameter when using Tesseract in cmd)
- OCR engine (--oem parameter when using Tesseract in cmd)
- ....

Thank you very much in advance

**Comment by JorjMcKie** [comment]
Thanks for your thoughts.
These things already are on our medium-term enhancements list.
